source/code-generation/query-converter/convert-views.txt

```diff
@@ -5,7 +5,193 @@ Convert Views
 =============
 
 .. contents:: On this page
-   :local:
-   :backlinks: none
-   :depth: 1
-   :class: singlecol
+  :local:
+  :backlinks: none
+  :depth: 1
+  :class: singlecol
+
+You can convert SQL views to MongoDB code with the query
+converter. The query converter considers the mapping rules and schema
+transformations defined in your project when converting your SQL code.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
+
+Before you Begin
+----------------
+
+.. include:: /includes/fact-query-converter-disclaimer.rst
+
+Steps
+-----
+.. procedure::
+  :style: normal
+
+  .. step:: Navigate to the query converter pane
+
+     From the :guilabel:`Code Generation` tab, click the
+     :guilabel:`Query Converter` pane.
+
+  .. step:: Open the query converter view
+
+     - If it is your first time using the query converter in your
+       project, click :guilabel:`Import From Database`.
+
+     - If your project already has converted SQL code, click the
+       :guilabel:`Manage Database Objects` button on the left-hand
+       pane.
+
+  .. step:: Connect to your relational database
+
+     a. Specify the connection details to your source database. 
+
+        For details, see :ref:`rm-relational-database-connection-strings`.
+
+     #. Click :guilabel:`Connect`.
+ 
+  .. step:: Select stored views
+
+     a. On the :guilabel:`Import Database Objects` modal, click
+        :guilabel:`Database`.
+     #. Click the :icon-fa5:`chevron-right` icon next to your schema.
+     #. Click the :icon-fa5:`chevron-right` icon next to
+        :guilabel:`Views`.
+     #. You can toggle views for conversion by clicking
+        the :icon-fa5:`check-square` icon next to a view's
+        name. All views are selected for conversion by default.
```

Comment:
Can we make this step more active voice?

To select the views to convert, click the (...). By default, all views are selected.